src/code/z_sram.c

```diff
@@ -339,9 +382,22 @@ void Sram_InitDebugSave(void) {
 }
 
 static s16 sDungeonEntrances[] = {
-    ENTR_YDAN_0,      ENTR_DDAN_0,      ENTR_BDAN_0,         ENTR_BMORI1_0,           ENTR_HIDAN_0, ENTR_MIZUSIN_0,
-    ENTR_JYASINZOU_0, ENTR_HAKADAN_0,   ENTR_HAKADANCH_0,    ENTR_ICE_DOUKUTO_0,      ENTR_GANON_0, ENTR_MEN_0,
-    ENTR_GERUDOWAY_0, ENTR_GANONTIKA_0, ENTR_GANON_SONOGO_0, ENTR_GANONTIKA_SONOGO_0,
+    ENTR_YDAN_0,             // SCENE_YDAN
+    ENTR_DDAN_0,             // SCENE_DDAN
+    ENTR_BDAN_0,             // SCENE_BDAN
+    ENTR_BMORI1_0,           // SCENE_BMORI1
```

Comment:
this one seems overkill cause the scene name is already there (and will always be, even if they change name), but yea I guess I dont care that much

Reply:
I'm thinking it makes it clear the scene id indexes this array, in case you're looking for uses of a specific scene id maybe or looking for what the scene id for deku does, or if you shift the scene table, want to add a dungeon...